manifests/client/install.pp

```diff
@@ -5,7 +5,19 @@
     fail 'class ::openldap::client has not been evaluated'
   }
 
-  package { $::openldap::client::package:
-    ensure => present,
+  case $::osfamily {
+    'Debian': {
+      package { $::openldap::client::package:
+        ensure => present,
+      }
+    }
+    'RedHat': {
+      class { '::openldap::client::utils':
```

Comment:
@werekraken we may need to add a `$utils_package` parameter in `params`, expose it to `init` and use it as default in `openldap::client::utils` instead.